features/cli/add_credential.feature

```diff
@@ -0,0 +1,65 @@
+Feature: Storing AWS access credentials
+
+  In order to manage AWS access credentials
+  As an AWS IAM user
+  I want to store an AWS access credential
+
+  Scenario: Adding a long-term access credential without an MFA device
+    When I run `aws-session add credential` interactively
+    And I type "AKIAXXXXXXXXXXXX"
+    And I type "secretaccesskey1234567890"
+    And I type ""
+    And I type "default"
+    Then the exit status should be 0
+    And the stdout should contain "AWS Secret Access Key"
+    And the stdout should contain "AWS Secret Access Key"
+    And the stdout should contain "MFA device (leave blank for none)"
+    And the stdout should contain "Credential alias"
+
+  Scenario: Adding a temporary security credential without an MFA device
+    When I run `aws-session add credential` interactively
```

Comment:
> You can optionally include multi-factor authentication (MFA) information when you call assume-role . This is useful for cross-account scenarios in which you want to make sure that the user who is assuming the role has been authenticated using an AWS MFA device.

http://docs.aws.amazon.com/cli/latest/reference/sts/assume-role.html

If I understand the exert above, then it is possible one can be required to supply an MFA token code when generating a session token for an IAM user and be required again to supply an MFA token code when assuming a role.